inc/stream/namespace.php

```diff
@@ -88,3 +89,15 @@ function override_network_admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
 		]
 	);
 }
+
+/**
+ * Add the Xray ID to the log item meta data.
+ *
+ * @param $record
+ *
+ * @return mixed
```

Comment:
The function itself seems to assume that we receive an array and also return array so we should update the type hinting on the function and this `@return` doc accordingly or add handling for any other types of variable that could come through.